test/test_jit_llga_fuser.py

```diff
@@ -1,51 +1,108 @@
 # Owner(s): ["module: mkldnn"]
+import sys
 import torch
 import unittest
 import itertools
 
 import torch.nn as nn
 import torch.nn.functional as F
+import torch.fx.experimental.optimization as optimization
 from torch.testing._internal.jit_utils import JitTestCase
 from torch.testing._internal.common_utils import run_tests, TEST_SCIPY, IS_WINDOWS, IS_MACOS
+from torch.testing._internal.common_device_type import (
+    instantiate_device_type_tests,
+    onlyCPU,
+    dtypes
+)
+
+def is_avx512_supported():
+    if sys.platform != 'linux':
+        return False
+    with open("/proc/cpuinfo", encoding="ascii") as f:
+        lines = f.read()
+    return "avx512" in lines
+
+IS_AVX512_UNSUPPORTED = not is_avx512_supported()
```

Comment:
@chunyuan-w, reusing `has_bf16_support` from `test_mkldnn.py` resulted in 2 CI checks failing (1 rocm & 1 CUDA),
so I readded this one instead, as all CI checks had passed with this approach yesterday. Thanks!